README.md

```diff
@@ -225,8 +225,8 @@ end
 Route patterns may have optional parameters:
 
 ``` ruby
-get '/posts.?:format?' do
-  # matches "GET /posts" and any extension "GET /posts.json", "GET /posts.xml" etc.
+get '/post/:format?' do
+  # matches "GET /posts" and any extension "GET /posts/json", "GET /posts/xml" etc
```

Comment:
Oh, wow. I didn't realize that nginx/apache would treat these differently. Yeah, that is tricky :/

Thin behaves the same way as webrick. I didn't check puma and unicorn.